### PR TITLE
Add some capabilities to core-chart, minor fix

### PR DESCRIPTION
**Pull request by hprommer**
core:
- contoller, pvc: add possibility to use existing PVC
- enforcer: enable configurable update-strategy
- all pods: add podLabels and podAnnotaitons

### CODE REVIEW

**Comment by becitsthere** [comment]
Can you remove the changes that modify the chart version?

**Comment by hprommer** [comment]
Version changes removed

File: charts/core/templates/controller-deployment.yaml [hunk 1]
```diff
@@ -199,7 +206,7 @@ apiVersion: policy/v1beta1
 kind: PodDisruptionBudget
 metadata:
   name: neuvector-controller-pdb
-  namespace: neuvector
```
Comment (becitsthere) [inline charts/core/templates/controller-deployment.yaml:202]: This piece has been modified. Can you remove it?

Comment (hprommer) [reply charts/core/templates/controller-deployment.yaml]: I reverted the change as desired. Still I do not understand, how the PDB should be working, if the helm-chart is installed in any other namespace than 'neuvector'

**Pull request merged**